packages/core/src/extensions/SuggestionMenu/defaultSlashMenuItems.ts

```diff
@@ -76,18 +77,21 @@ export function insertOrUpdateBlock<
   return insertedBlock;
 }
 
-export function getDefaultSlashMenuItems() {
+export function getDefaultSlashMenuItems<
```

Comment:
does this type change still work if you want to add a custom block (like `insertAlert`)?

Reply:
Will change it so that only schema compatible items are returned so that you can use it for any schema. That will also make it more inline with how stuff in the formatting toolbar works.